opre_ops/ops_site/helpers/csv_to_json.py

```diff
@@ -0,0 +1,44 @@
+import csv
+import json
+import argparse
+
+
+def convert_csv_to_json(inputf, outputf, keys, model_name):
```

Comment:
_suggestion (non-blocking):_ 

It might be helpful to add a few code comments here to document:

1. the CLI command you use to run this script, so that other devs know how to use it
2. a quick note about the purpose of this script ("load in fake/sample data for development purposes")

Reply:
Added a docstring, lmk what you think!

Reply:
Looks great, thanks Carly!